File: packages/scanner/tests/fixtures/VCS-015/vulnerable.py
import os
import subprocess

def deploy(branch):
    subprocess.run(f"git pull origin {branch}", shell=True)

def list_files(path):
    return os.system(f"ls -la {path}")

def run_pipe(cmd):
    return os.popen(cmd).read()

def parse_config(text):
    return eval(text)

def execute_block(code):
    exec(code)

File: packages/scanner/tests/fixtures/VCS-015/vulnerable.js
const { exec, execSync } = require("child_process");
const vm = require("vm");

function deploy(branch) {
  exec(`git pull origin ${branch}`);
}

function listFiles(path) {
  return execSync(`ls -la ${path}`).toString();
}

function parseExpr(input) {
  return eval(input);
}

function runIsolated(code) {
  return vm.runInNewContext(code);
}

function fnFromString(body) {
  return new Function(body);
}
